apps/labs/labs-louisehuyton-nodejs/labs-louisehuyton-nodejs.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: labs-louisehuyton-nodejs
+  namespace: labs
+spec:
+  releaseName: labs-louisehuyton-nodejs
+  chart:
+    spec:
+      chart: ./stable/labs-louisehuyton-nodejs
+      sourceRef:
+        kind: GitRepository
+        name: hmcts-charts
+        namespace: flux-system
+      interval: 1m
+  values:
+    nodejs:
+      image: hmctssandbox.azurecr.io/labs/louisehuyton-nodejs:latest # {"$imagepolicy": "flux-system:labs-louisehuyton-nodejs"}
+      disableTraefikTls: true
+      environment:
+        RECIPE_BACKEND_URL: http://labs-louisehuyton-nodejs-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```

Comment:
where is this coming from? you don't need RECIPE_BACKEND_URL

Reply:
will have a look